### PR TITLE
Add Configuration and Install pages

### PR DESCRIPTION
**Pull request by qu0b**

### CODE REVIEW

**Comment by qu0b** [comment]
I've updated the dialog

File: src/react/components/ImportKeystore.tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
+import { BackgroundLight, } from '../colors';
+import {  Button, FormControl, FormControlLabel, Radio, RadioGroup, Typography, Box, Grid, Modal, InputAdornment, TextField, styled, OutlinedInputProps, TextFieldProps } from '@mui/material';
+import React, { FC, ChangeEvent, Dispatch, ReactElement, SetStateAction, useState, MutableRefObject } from 'react';
+
+import { FileCopy, LockOpen } from '@mui/icons-material'
+import { Network } from '../types';
```
[the remark this answers is not in the document]
Comment (remyroy) [reply src/react/components/ImportKeystore.tsx:6]: and could be removed

File: src/react/pages/SystemOverview.tsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { Grid, Typography } from '@mui/material';
 import { Network } from "../types";
```
Comment (remyroy) [inline src/react/pages/SystemOverview.tsx:3]: Unused import. Could be removed.

**Review by remyroy — CHANGES_REQUESTED** [review]
Some changes are needed. I could do them myself if you want.

File: src/react/components/ImportKeystore.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (qu0b) [reply src/react/components/ImportKeystore.tsx]: fixed

File: src/react/components/ImportKeystore.tsx [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+  setKeystorePassword: Dispatch<SetStateAction<string>>
+  closing: MutableRefObject<((arg: () => Promise<boolean>) => void) | undefined>
+  installationDetails: InstallDetails
+
+
+//   setInstallationDetails: Dispatch<SetStateAction<InstallDetails>>,
+//   installationDetails: InstallDetails,
+}
+
+// console.log(dialog.showOpenDialog({ properties: ['openFile'] }))
```
[the remark this answers is not in the document]
Comment (qu0b) [reply src/react/components/ImportKeystore.tsx]: fixed

File: src/react/constants.ts [hunk 4]
```diff
@@ -11,3 +12,39 @@ export const stepLabels = {
...
+	key: ConsensusClient.NIMBUS,
+	label: 'Nimbus',
+}, {
+	key: ConsensusClient.TEKU,
+	label: 'Teku'
+}
+]
+export const ExecutionClients: IExecutionClient[] = [{
+	key: ExecutionClient.GETH,
+	label: 'Geth (recommended)',
```
[the remark this answers is not in the document]
Comment (qu0b) [reply src/react/constants.ts]: I could implement a random order for the next release

**Comment by qu0b** [comment]
I think I resolved all of the changes, could you make sure that it still runs in your VM before releasing it?

**Comment by remyroy** [comment]
Everything is good. Merging.

**Pull request merged**